Feedback on Docs and community sections on Docsy theme

Dindihub opened the pull request:
Docs and Community sections completed according to recommendations from the TUF analysis. 

I would appreciate your feedback @lukpueh and @chalin

The home page, Resources,  and whole website styling are still in progress. Finishing the content structure and then getting right back to it :)

Thanks

Dindihub committed:
Committing local changes before merge

Dindihub committed:
Merge remote-tracking branch 'upstream/main'

Dindihub committed:
update from upstream

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Edit home page

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Populate docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Edit page descriptions

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Edit page descriptions

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Restructure Docs menu

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Restructure docs/projects section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Add community section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Structure project section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Correct typos

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Change icon color on community page

Signed-off-by: sandra <sadindi03@gmail.com>

chalin commented on yarn.lock:
Don't commit lock files.
Remove this file, and add it to `.gitignore`.
Also, use NPM rather than Yarn.

Dindihub replied:
Done

chalin commented on assets/scss/_variables_project.scss:
```diff
@@ -1 +1,5 @@
 /* Add styles or override variables from the theme here. */
+
+.hero-comm{
+background-color: #0082ca;
+}
```
This variables file should be used only for variables.
Move this style definition to the project's style file. See the in-toto site that @Dariksha is working on or the OpenTelemetry.io repo for examples.

Dindihub replied:
@chalin Thanks for the clarification. I was trying to change the background color of the hero section on the community page. Now I see why it was not working :)

Dindihub committed:
Update README.md

Dindihub committed:
Merge branch 'main' into TUFwebsite

Dindihub committed:
restructure docs section

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Merge branch 'TUFwebsite' of https://github.com/Dindihub/docsy-starter into TUFwebsite

Dindihub committed:
description of Adoptions page

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
description of Adoptions page

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
reduce hero-image

Signed-off-by: sandra <sadindi03@gmail.com>

chalin commented on content/en/docs/faq/faq.md:
```diff
@@ -0,0 +1,163 @@
+---
+title: FAQ
+weight: 45
+# aliases: [/docs/faq/faq]
```
```suggestion
```

chalin commented on content/en/docs/faq/faq.md:
```diff
@@ -0,0 +1,163 @@
+title: FAQ
+weight: 45
+# aliases: [/docs/faq/faq]
+description: Frequently Asked Questions
```
```suggestion
```

chalin commented on content/en/docs/faq/faq.md:
```diff
@@ -0,0 +1,163 @@
+---
+title: FAQ
```
```suggestion
title: Frequently Asked Questions
LinkTitle: FAQ
```

chalin commented on content/en/docs/faq/faq.md:
```diff
@@ -0,0 +1,163 @@
+# aliases: [/docs/faq/faq]
+description: Frequently Asked Questions
+---
+
```
```suggestion
```

chalin commented on content/en/docs/faq/faq.md:
- This file should be moved up one level (to sit directly under `docs`), and the `faq` folder deleted.
- Do the same for all other folder-filename pairs under `docs` of the form `docs/XYZ/XYZ.md`, where the folder `XYZ` contains only that single page.

chalin commented on content/en/herobackground.jpg:
I'd suggest that we keep the name `content/en/featured-background.jpg`.

chalin commented on content/en/resources/_index.md:
```diff
@@ -0,0 +1,8 @@
+title: Resources
+menu: {main: {weight: 50}}
+description: Learn more from our resources
+
```
Avoid blank lines in the the page front matter, in this file and all other files:

```suggestion
```

chalin commented on content/en/resources/publications/publications.md:
```diff
@@ -1,9 +1,15 @@
+
 ---
 
+
+
+
+
+
```
```suggestion
```

chalin reviewed (commented):
See inline comments.

lukpueh commented on content/en/_index.md:
```diff
@@ -1,16 +1,9 @@
 ---
 
-{{% blocks/cover title="Docsy Starter" image_anchor="top" height="full" %}}
+{{% blocks/cover title="The Update Framework"image_anchor="top" color="primary" height="max" %}}
```
```suggestion
{{% blocks/cover title="The Update Framework" image_anchor="top" color="primary" height="max" %}}
```

lukpueh commented on content/en/_index.md:
```diff
@@ -19,33 +12,59 @@ show_banner: true
+{{% blocks/feature icon="fa-lightbulb" title="Our work" url="docs/" %}}
+
+This material is based upon work supported by the National Science Foundation under Grant Nos. CNS-1345049 and CNS-0959138. Any opinions, findings, and conclusions or recommendations expressed in this material are those of the author(s) and do not necessarily reflect the views of the National Science Foundation.
+
```
IMO this texts fits better into an acknowledgment footer. Here I'd add something that encourages people to  "Read more" in the Docs subsection.

lukpueh commented on content/en/_index.md:
```diff
@@ -19,33 +12,59 @@ show_banner: true
 
-{{% blocks/feature icon="fa-brands fa-x-twitter" title="Follow us on Twitter!" url="https://twitter.com/docsydocs" %}}
-Find out about new features and how our users are using Docsy.
+{{% blocks/feature icon="fa-brands fa-x-twitter" title="Contact Us" %}}
```
Let's not use the Twitter icon. I don't think we even are on twitter. Maybe there's a generic "Contact" icon. You can search here: https://fontawesome.com/

lukpueh reviewed (commented):
This is great. I particularly like the "Get started" page. 💯 

Some high-level contents regarding structure and content of the Docs section:

- Let's move Timeline and History after Get Started. They seem like secondary interest. 
- What do you think about making Project and Security subsections of Overview, or consolidate it otherwise?
- The tag line for "Security" -- "Conduct security audits with TUF" -- is not accurate. The page summarises the security properties of a hypothetical TUF repository, which is only very remotely related to security audits.
-  Metadata shouldn't be a subsection of Project. Its purpose is to better understand how TUF works and not how the TUF project is structured. IMO it could be under Overview or Get started.
- Maybe the TAPs subsection doesn't need to be its own page, but could be integrated directly with the project page.
- Examples seems to be left-over docsy boilerplate. Please remove.

Dindihub committed:
Update content/en/docs/faq/faq.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/docs/faq/faq.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/docs/faq/faq.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
remove folders from subsections under Docs folder

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Fix broken links in docs

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Update content/en/resources/publications/publications.md

Co-authored-by: Patrice Chalin <chalin@users.noreply.github.com>

Dindihub committed:
Update content/en/_index.md

Co-authored-by: Lukas Pühringer <luk.puehringer@gmail.com>

Dindihub committed:
Realign docs section structure

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Fix broken links

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub committed:
Change icons on homepage and delete styles from scss file

Signed-off-by: sandra <sadindi03@gmail.com>

Dindihub commented:
> @Dindihub - after you address @lukpueh and my latest comments, I'd suggest that we merge this PR and work incrementally from there.
> 
> WDYT @lukpueh?

I have addressed all the comments.  I'm am okay to merge.

Dindihub committed:
Edit content on home page

Signed-off-by: sandra <sadindi03@gmail.com>

chalin reviewed (approved):
Very nice incremental improvement. LGTM. I'll followup with issues for further changes.
